modules/op-release-notes-1-0.adoc

```diff
@@ -72,6 +72,38 @@ For this release, use flags `disable-home-env-overwrite` and `disable-working-di
 == Known issues
 * If you are upgrading from an older version of {pipelines-title}, you must delete your existing deployments before upgrading to {pipelines-title} version 1.0. To delete an existing deployment, you must first delete Custom Resources and then uninstall the {pipelines-title} Operator. For more details, see the uninstalling {pipelines-title} section.
 * Submitting the same `v1alpha1` Tasks more than once results in an error. Use `oc replace` instead of `oc apply` when re-submitting a `v1alpha1` Task.
+* `buildah` ClusterTask does not work on the {pipelines-title} version 1.0.
```

Comment:
```suggestion
* `buildah` ClusterTask does not work on the {pipelines-title} version 1.0 when adding a new user inside a container.
```

Reply:
How about - The `buildah` ClusterTask does not work when a new user is added to a container.

Reply:
👍